CODE_OF_CONDUCT.md

```
@@ -0,0 +1,23 @@
+<div align="center" width="100%"><img src="./assets/images/open-communities.png"></div>
```

Markdownlint

Element: div
Inline HTML

(MD033, no-inline-html)